terminology.md

```diff
@@ -452,6 +452,41 @@ Another example is whether an ECMAScript source file will be executed as a scrip
 
 [Import map proposal](https://github.com/WICG/import-maps#supplying-out-of-band-metadata-for-each-module)
 
+## Override mistake
+
+### Definition
+
+The behavior that ECMAScript throws a TypeError (in the strict mode) for code `O[K] = ...` when `K` is a non-writable property in the prototype of `O`.
+
+This behavior brings serious compatibility problems in the ecosystem when the runtime is "frozen" (which means all built-in APIs are frozen) for security or performance reasons.
```

Comment:
I think we've had more compat problems with new properties than with frozen intrinsics, actually. Might as well mention both:

```suggestion
This behavior brings serious compatibility problems in the ecosystem when any properties are non-writable, including when [the language adds new built-in non-writable properties to existing prototypes](https://github.com/tc39/proposal-iterator-helpers/issues/115) and when [the runtime or code on the page freezes built-ins](https://github.com/tc39/proposal-iterator-helpers/issues/286).
```

Reply:
oh we're very unlucky, I don't know iterator helper hit both of them